### DISCUSSION TITLE
Question: how to track how many users are using PWA with google analytics?

### DISCUSSION THREAD

**Discussion by meotimdihia** [shadowwalker/next-pwa#193] [Q&A]
Tried to search around about this problem. But I still don't know how to make it works. Anyone give tips for me on how to implement it.

**Comment by brianshano** [comment]
In your manifest.json add a param to your start_url so you can track it in GA. 
`  "start_url": "/?=pwa",`

**Reply by meotimdihia** [reply]
Thanks, I'm using 
`
  "start_url": "/?utm_source=web_app_manifest",
`
And I saw stats in `Traffic Acquisition`. I don't try your param yet.
